WBooks/Repositories/BookRepository.swift

```diff
@@ -28,8 +30,49 @@ class BookRepository {
                 }
             }
     }
+    
+    public func rentBook(bookId: Int, onSuccess: @escaping () -> Void, onError: @escaping (Error) -> Void) {
+        let endpoint = "\(baseUrl)/users/5/rents"
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        let params: [String: Any] = [
+            "userID": 5,
+            "bookID": bookId,
+            "from": dateFormatter.string(from: Date()),
+            "to": dateFormatter.string(from: Calendar.current.date(byAdding: .day, value: 1, to: Date()) ?? Date())
```

Comment:
This is business logic, so it must be in the viewModel and get them here as params, you could have a model if you want